Tests/L2Tests/L2Tests/CMakeLists.txt

```diff
@@ -21,12 +21,23 @@ set(MODULE_NAME ${NAMESPACE}${PLUGIN_NAME})
 set(THUNDER_PORT 9998)
 find_package(${NAMESPACE}Plugins REQUIRED)
 
+include_directories(
```

Comment:
What is the need for adding these? How SystemServices L2 test working with out adding like this?